libs/designer-ui/src/lib/dynamicallyaddedparameter/helper.ts

```diff
@@ -1,11 +1,7 @@
-import type { DynamicallyAddedParameterProps, DynamicallyAddedParameterTypeType, IDynamicallyAddedParameterProperties } from '.';
-import { DynamicallyAddedParameterType } from '.';
-import constants from '../constants';
-import type { ValueSegment } from '../editor';
-import { ValueSegmentType } from '../editor';
+import type { DynamicallyAddedParameterTypeType } from '../dynamicallyaddedparameter';
```

Comment:
Most of the code here moved to `libs/designer-ui/src/lib/floatingactionmenu/floatingactionmenuinputs/helper.ts`.

Only notable small change is to `generateDynamicParameterKey()`